implementation/contracts/system/TBTCToken.sol

```diff
@@ -30,7 +28,7 @@ contract TBTCToken is ERC20Detailed, ERC20, VendingMachineAuthority {
         return true;
     }
 
-    /// @dev             Burns an amount of the token of a given account
+    /// @dev             Burns an amount of the token of a given account.
```

Comment:
I would say “from the given account's balance”, maybe? This phrasing is a little weird.